devtools/cli/src/main/java/io/quarkus/cli/update/RewriteGroup.java

```diff
@@ -5,11 +5,12 @@
 public class RewriteGroup {
 
     @CommandLine.Option(order = 0, names = {
-            "--no-rewrite" }, description = "Disable the rewrite feature.", defaultValue = "false")
-    public boolean noRewrite = false;
+            "--yes", "-y",
+            "--run" }, description = "Run the suggested update recipe for this project using OpenRewrite.", defaultValue = "false")
+    public boolean yes = false;
 
     @CommandLine.Option(order = 1, names = {
-            "--dry-run" }, description = "Rewrite in dry-mode.", defaultValue = "false")
+            "--dry-run" }, description = "Run the suggested update recipe for this project using OpenRewrite in dry-mode (no changes).", defaultValue = "false")
```

Comment:
I wouldn't push "OpenRewrite" in the help here - that is an implementation detail IMO.

```suggestion
            "--dry-run" }, description = "Do a dry run of the suggested update recipe for this project.", defaultValue = "false")
```